### DISCUSSION TITLE
Adding feeds from live WordPress site to staging site

### DISCUSSION THREAD

**Discussion by fritzisa** [code4recovery/12-step-meeting-list#642] [Q&A] [answered]
Our current Area 59 site (area59aa.org) includes feeds from a dozen or so Districts and Intergroups. It's not apparent to me how I can add those feeds to our staging site, which is nearing go-live:
https://rps.866.myftpupload.com/find-a-meeting/

The content currently used in the staging site is a JSON export from the live site.

**Comment by kiyote33** [comment, marked as answer]
Hi @fritzisa,
A simple way to populate your staging site with the contents from your area59aa.org site is to use the export CSV feature found under the "Where's My Info?" section on the Import & Settings page. The generated meetings.csv file will download to your local machine and you can then use it with the "Import CSV" feature to populate your new staging site.
Hope this helps...

**Reply by fritzisa** [reply]
Thank you, that's what I've done already. But how can I get those feeds into my staging site, so that when I go live the feeds will be there to refresh daily.

**Reply by kiyote33** [reply]
Another option is to set the Sharing option to "Open". That would allow you to use the Data Source import feed (i.e... https://area59aa.org/wp-admin/admin-ajax.php?action=meetings) and refresh the data source from time to time.
